guides/common/modules/con_security-content-automation-protocol.adoc

```diff
@@ -1,9 +1,31 @@
 [id="Security_Content_Automation_Protocol_{context}"]
 = Security Content Automation Protocol
 
-{Project} uses the Security Content Automation Protocol (SCAP) to define security configuration policies.
-For example, a security policy might specify that for hosts running {RHEL}, login via SSH is not permitted for the `root` account.
-With {Project}, you can schedule compliance auditing and reporting on all managed hosts.
-ifndef::orcharhino[]
-For more information about SCAP, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/security_hardening/[_{RHEL}{nbsp}8 Security hardening_].
-endif::[]
+{Project} uses the Security Content Automation Protocol (SCAP) standard to define security policies.
+
+SCAP is a framework of several specifications based on XML, such as checklists described in the Extensible Checklist Configuration Description Format (XCCDF) and vulnerabilities described in the Open Vulnerability and Assessment Language (OVAL).
+The specifications can be encapsulated as _data stream_ files.
+
+Checklist items in XCCDF, also known as _rules_, express the desired configuration of a system item.
+For example, you may specify that no one can log in to a host over SSH using the `root` user account.
+Rules can be grouped into one or more _XCCDF profiles_, allowing multiple profiles to share a rule.
+
+An OpenSCAP scanner evaluates system items on a host against the rules and generates a report in the Asset Reporting Format (ARF), which is then returned to {Project} for monitoring and analysis.
+
+.Specifications in the SCAP Framework 1.3
+[cols="1,4,1"]
+|===
+|*Title* |*Description* |*Version*
+|SCAP |Security Content Automation Protocol |1.3
+|XCCDF |Extensible Configuration Checklist Description Format |1.2
+|OVAL |Open Vulnerability and Assessment Language |5.11
+|- |Asset Identification |1.1
```

Comment:
I am confused as to why this item has no title.

Reply:
This spec doesn't have an abbreviation, I don't know why, I was confused too :D